registrations/views.py

```diff
@@ -233,8 +233,15 @@ def post(self, request):
 
         post_data = self.build_jembi_helpdesk_json(serializer.validated_data)
         try:
+
+            source = Source.objects.get(user=self.request.user.id)
+
+            endpoint = 'helpdesk'
+            if source.name == 'nc_helpdesk':
```

Comment:
Looking at the existing source names in production, I wonder if we can make this more like that format to keep things consistent? 
![image](https://user-images.githubusercontent.com/8234653/32829518-f9791fd8-c9fa-11e7-80f9-486a05e1e170.png)